<launch>
	<node name="motor_comm" pkg="shared_serial" type="server">
		<param name="port_name" value="/dev/ttyUSB0"/>
		<param name="port_type" value="RS485_FTDI"/>
		<param name="baud_rate" value="921600"/>
	</node>
</launch>
